packages/molecule/src/grammar/grammar.js

```diff
@@ -0,0 +1,1980 @@
+// @generated by Peggy 4.0.3.
```

Comment:
This file should not be included in git, but generated when the package is being published.

Reply:
The codegen file is removed and ignored. It will be generated when running `npm run build`